src/js/pages/Home.jsx

```diff
@@ -1,12 +1,14 @@
 import React, { useState, useEffect } from 'react';
 
 import Pipeline from 'js/components/pipeline/Pipeline';
+import PullRequests from 'js/components/pipeline/PullRequests';
 import Page from 'js/pages/templates/Page';
 
-import { getPipelineDataInitial, getPipelineDataAPI, getUser } from 'js/services/api';
+import { getPipelineDataInitial, getPipelineDataAPI, getPRs } from 'js/services/api';
 
 export default () => {
   const [pipelineState, setPipelineData] = useState(getPipelineDataInitial());
+  const [prsState, _] = useState(getPRs);
```

Comment:
Shouldn't this be:
```suggestion
  const [prsState, _] = useState(getPRs());
```
?

Reply:
Absolutely. I guess if it locally works bec React just call it if a function is retrieved. Anyhow, calling it explicitly.